imap/promstatsd.c

```diff
@@ -84,12 +85,16 @@ static struct report {
     int frequency;
     int64_t prev_report_time;
 } reports[] = {
-    { FNAME_PROM_SERVICE_REPORT, "service", &do_collate_service_report,
-      IMAPOPT_PROMETHEUS_SERVICE_UPDATE_FREQ, 10,
-      NULL, BUF_INITIALIZER, 0, 0 },
-    { FNAME_PROM_USAGE_REPORT,   "usage",   &do_collate_usage_report,
-      IMAPOPT_PROMETHEUS_USAGE_UPDATE_FREQ, 0,
-      NULL, BUF_INITIALIZER, 0, 0 },
+    { FNAME_PROM_SERVICE_REPORT,
+     "service", &do_collate_service_report,
+     IMAPOPT_PROMETHEUS_SERVICE_UPDATE_FREQ, 10,
+     NULL, BUF_INITIALIZER,
+     0, 0 },
+    { FNAME_PROM_USAGE_REPORT,
+     "usage",   &do_collate_usage_report,
+     IMAPOPT_PROMETHEUS_USAGE_UPDATE_FREQ,   0,
+     NULL, BUF_INITIALIZER,
+     0, 0 },
```

Comment:
I don't like what it's done here.  The careful grouping of related fields has been replaced with seemingly random line breaks based on... line length?  But then why are the final `0, 0` always on their own line, instead of grouped with the line above that they would easily fit on?  The wrapping used to communicate something, now it's just confusing.